### PR TITLE
Fix button outlines (in more places) and initially set focus on back button when selecting region clipping

### PR DESCRIPTION
**Pull request by jogonzal**
Fix for https://office.visualstudio.com/OneNote/_workitems/edit/3177675

### CODE REVIEW

File: src/scripts/clipperUI/panels/dialogPanel.tsx [hunk 1]
```diff
@@ -55,14 +55,12 @@ export abstract class DialogPanelClass extends ComponentBase<{}, DialogPanelProp
...
 							return (
-								<a id={button.id} className="dialogButton" {...this.enableInvoke({callback: button.handler, tabIndex: 70})}>
-									<div className="wideButtonContainer">
-										<a className="wideButtonFont wideActionButton"										
-											style={Localization.getFontFamilyAsStyle(buttonFontFamily)}>
-											{ button.label }
-										</a>
-									</div>
-								</a>
+								<div id={button.id} className="wideButtonContainer dialogButton" {...this.enableInvoke({callback: button.handler, tabIndex: 70})}>
```
Comment (mttwc) [inline src/scripts/clipperUI/panels/dialogPanel.tsx]: Would this break tabbing+enter interactions now that enableInvoke is living on the outer div and not the a?

File: src/scripts/clipperUI/panels/regionSelectingPanel.tsx [hunk 2]
```diff
@@ -20,8 +24,8 @@ class RegionSelectingPanelClass extends ComponentBase<{}, ClipperStateProp> {
 							{Localization.getLocalizedString("WebClipper.Label.DragAndRelease")}
 						</span>
 					</div>
-					<div className="wideButtonContainer">
-						<a id={Constants.Ids.regionClipCancelButton} role="button" {...this.enableInvoke({callback: this.handleCancelButton, tabIndex: 70})}>
+					<div className="wideButtonContainer" {...this.onElementFirstDraw(this.initiallySetFocusToBackButton)} {...this.enableInvoke({callback: this.handleCancelButton, tabIndex: 0})}>
```
Comment (mttwc) [inline src/scripts/clipperUI/panels/regionSelectingPanel.tsx]: See above comment

File: src/scripts/clipperUI/panels/signInPanel.tsx [hunk 3]
```diff
@@ -172,22 +172,18 @@ class SignInPanelClass extends ComponentBase<SignInPanelState, SignInPanelProps>
...
-					</a>
-					<a id={Constants.Ids.signInButtonOrgId} role="button" {...this.enableInvoke({callback: this.onSignInOrgId, tabIndex: 12, args: AuthType.OrgId})}>
-						<div className="wideButtonContainer">
-							<span className="wideButtonFont wideActionButton"
-								style={Localization.getFontFamilyAsStyle(Localization.FontFamily.Regular) }>
-								{Localization.getLocalizedString("WebClipper.Action.SigninOrgId") }
-							</span>
-						</div>
-					</a>
+					<div className="wideButtonContainer" id={Constants.Ids.signInButtonMsa} role="button" {...this.enableInvoke({callback: this.onSignInMsa, tabIndex: 11, args: AuthType.Msa})}>
```
Comment (mttwc) [inline src/scripts/clipperUI/panels/signInPanel.tsx]: See above comment for these two

File: src/scripts/clipperUI/panels/successPanel.tsx [hunk 4]
```diff
@@ -36,14 +36,12 @@ class SuccessPanelClass extends ComponentBase<{ }, ClipperStateProp> {
...
 				</div>
-				<a id={Constants.Ids.launchOneNoteButton} {...this.enableInvoke({callback: this.onLaunchOneNoteButton, tabIndex: 70})}>
-					<div className="wideButtonContainer">
-						<span className="wideButtonFont wideActionButton"
-							style={Localization.getFontFamilyAsStyle(Localization.FontFamily.Regular)}>
-							{Localization.getLocalizedString("WebClipper.Action.ViewInOneNote")}
-						</span>
-					</div>
-				</a>
+				<div id={Constants.Ids.launchOneNoteButton} className="wideButtonContainer" {...this.enableInvoke({callback: this.onLaunchOneNoteButton, tabIndex: 70})}>
```
Comment (mttwc) [inline src/scripts/clipperUI/panels/successPanel.tsx]: See above comment

**Review by mttwc — COMMENTED** [review]
Regarding the button outlines, was this broken on a specific set of browsers, or all of them?

File: src/scripts/clipperUI/panels/dialogPanel.tsx (hunk 1 above)
Comment (mttwc) [reply src/scripts/clipperUI/panels/dialogPanel.tsx]: Also wouldn't this break narrator?

Comment (jogonzal) [reply src/scripts/clipperUI/panels/dialogPanel.tsx]: This definitely did not break tabbing+enter interactions, but I will test with Edge + Narrator

**Review by mttwc — APPROVED** [review]

**Comment by jogonzal** [comment]
It was broken in all browsers for the "view in OneNote", "sign in with MS account" and "sign in with orgid account"

File: src/scripts/clipperUI/panels/dialogPanel.tsx (hunk 1 above)
Comment (jogonzal) [reply src/scripts/clipperUI/panels/dialogPanel.tsx]: This did break Edge + Narrator :(. Nice catch. I will go back to the "a"

**Review by mttwc — APPROVED** [review]

**Pull request merged**